app/Http/Controllers/V2/Dashboard/ProjectProfileDetailsController.php

```diff
@@ -3,32 +3,28 @@
 namespace App\Http\Controllers\V2\Dashboard;
 
 use App\Http\Controllers\Controller;
-use App\Http\Resources\V2\Dashboard\ProjectProfileDetailsResource;
 use App\Models\V2\Forms\FormOptionListOption;
 use App\Models\V2\Projects\Project;
 use Illuminate\Http\Request;
 
 class ProjectProfileDetailsController extends Controller
 {
-    public function __invoke(Request $request): ProjectProfileDetailsResource
+    public function __invoke(Request $request, string $project = null)
```

Comment:
`This should be `Project $project` instead of `string $project = null`. And then fetching the project on line 14 goes away.